src/AzureAuth/EnvVars.cs

```diff
@@ -18,6 +18,11 @@ internal static class EnvVars
         /// </summary>
         public static readonly string Config = $"{EnvVarPrefix}_CONFIG";
 
+        /// <summary>
+        /// The name of an environment variable used to override the cache file path.
+        /// </summary>
+        public static readonly string AZUREAUTH_CACHE_FILE = $"{EnvVarPrefix}_CACHE_FILE";

```

Comment:
Concistency - The `AZUREAUTH_CONFIG` value is for a file but does not use `FILE` in the name. So I'd prefer we name the env var: `AZUREAUTH_CACHE` to be consistent.

Reply:
Changed the Env Var to "AZUREAUTH_CACHE", also changed the variable name to Cache for consistency.